File: src/main/java/grp/faq/dao/GroupMapper.java
package grp.faq.dao;

import grp.faq.entity.Group;

public interface GroupMapper {
    int deleteByPrimaryKey(Long cGid);

    int insert(Group record);

    int insertSelective(Group record);

    Group selectByPrimaryKey(Long cGid);

    int updateByPrimaryKeySelective(Group record);

    int updateByPrimaryKey(Group record);
}

File: src/main/scala/grp/faq/dao/GroupMapper.scala
package grp.faq.dao

import grp.faq.entity.Group
import java.util.List

import org.apache.ibatis.annotations.Param

trait GroupMapper {
  def selectGroupById(@Param("gid")gid: Int): Group = ???

  def getAllGroup(): List[Group]

  def add(group: Group): Boolean
}
